test/extensions/repository_test.rb

```diff
@@ -112,15 +112,21 @@ def test_search_by_repository_ids
     end
 
     def test_create_or_update_schedule
-      response = @extension.create_or_update_schedule(RepositorySupport.repo_id, 'yum_importer', '2012-09-25T20:44:00Z/P7D')
+      response = @extension.create_or_update_schedule(RepositorySupport.repo_id,
+                                                      'yum_importer',
+                                                      '2012-09-25T20:44:00Z/P7D')
```

Comment:
This was a line length issue. I think the code you had would work. Let me update it and we'll see.